src/renderer/components/composer/menuAttachment.tsx

```diff
@@ -111,59 +102,50 @@ export default function MenuAttachment({
   ])
 
   // item array used to populate menu
-  const items: MenuAttachmentItemObject[] = [
+  const menu: (ContextMenuItem | false)[] = [
     settings?.settings.webrtc_instance && {
-      id: 0,
-      icon: 'phone' as IconName,
-      text: tx('videochat'),
-      onClick: onVideoChat,
+      icon: 'bell',
+      // icon: 'phone' as IconName,
+      label: tx('videochat'),
+      action: onVideoChat,
     },
     {
-      id: 1,
-      icon: 'media' as IconName,
-      text: tx('image'),
-      onClick: addFilenameMedia.bind(null),
+      icon: 'image',
+      // icon: 'media' as IconName,
+      label: tx('image'),
+      action: addFilenameMedia.bind(null),
     },
     {
-      id: 2,
-      icon: 'document' as IconName,
-      text: tx('file'),
-      onClick: addFilenameFile.bind(null),
+      icon: 'info',
+      // icon: 'document' as IconName,
+      label: tx('file'),
+      action: addFilenameFile.bind(null),
     },
-  ].filter(item => !!item) as MenuAttachmentItemObject[]
+  ]
 
-  return (
-    <div className='attachment-button'>
-      <Popover
-        content={
-          <Menu>
-            <MenuAttachmentItems itemsArray={items} />
-          </Menu>
-        }
-        position={Position.TOP_LEFT}
-      >
-        <Button id='test-attachment-menu' minimal icon='paperclip' />
-      </Popover>
-    </div>
-  )
-}
+  const onClickAttachmentMenu = (event: React.MouseEvent<any, MouseEvent>) => {
+    const threeDotButtonElement = document.querySelector(
+      '#attachment-menu-button'
+    ) as HTMLDivElement
+
+    const boundingBox = threeDotButtonElement.getBoundingClientRect()
+
+    const [cursorX, cursorY] = [boundingBox.x, boundingBox.y]
+    event.preventDefault() // prevent default runtime context menu from opening
+
+    openContextMenu({
+      cursorX,
+      cursorY,
+      items: menu,
+    })
+  }
 
-// Function to populate Menu
-const MenuAttachmentItems = ({
-  itemsArray,
-}: {
-  itemsArray: MenuAttachmentItemObject[]
-}) => {
   return (
-    <>
-      {itemsArray.map((item: MenuAttachmentItemObject) => (
-        <MenuItem
-          key={item.id}
-          icon={item.icon}
-          text={item.text}
-          onClick={item.onClick}
-        />
-      ))}
-    </>
+    <Button
```

Comment:
use a normal button element here and adjust styling with the `attachment-button` class.

we want to get rid of blueprintjs and it currently looks bad in dark theme:
<img width="106" alt="Bildschirmfoto 2024-05-10 um 04 15 42" src="https://github.com/deltachat/deltachat-desktop/assets/18725968/cc4c4b85-79a2-482f-babe-0d536cccac75">

(I know this is just a draft, but wanted to give that feedback already anyway)

Reply:
Pushed an update for this, there only paperclip styling left (change color and make it a bit smaller).

I made copies of image.svg for icons used before ('document', 'media', 'phone'), should I just copy old svg data there or we need to make new ones?

Reply:
just download icons from https://fonts.google.com/icons that look similar / have the same message if there are no matching icons in `images/icons` yet. `images/icons` is the folder for the icons.

Reply:
I think we should not keep using the blueprint icons unless we want to specify that they have a different license than the google/material-design ones.